vnext/template/cs-app/proj/Directory.Build.props

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+
+  <PropertyGroup Label="NuGet" Condition="'$(MSBuildProjectExtension)' == '.vcxproj'">
+    <AssetTargetFallback>$(AssetTargetFallback);native</AssetTargetFallback>
+    <TargetFrameworkMoniker>native,Version=v0.0</TargetFrameworkMoniker>
+  </PropertyGroup>
+
+  <ItemGroup Label="NuGet" Condition="'$(MSBuildProjectExtension)' == '.vcxproj'">
+    <ProjectCapability Include="PackageReferences" />
+  </ItemGroup>
+
+</Project>
```

Comment:
So that any VCXPROJ within that solution **can** (this doesn't make it mandatory to) use PackageReferences.

For clarity, did you mean "... adding this only for **C++** projects"?
If so, it's because C# Projects don't need those settings. They are meant to allow C++ projects to play along with .NET dependencies (packages).